src/main/java/eggventory/logic/parsers/ParseEdit.java

```diff
@@ -15,18 +15,20 @@ private Command processEditStock(String input) throws BadInputException {
 
         String stockCode = addInput[0];
```

Comment:
loll the variable name should be editInput technically hahahaha

Reply:
right..nice catch